app/public/index.html

```diff
@@ -19,6 +19,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
```

Comment:
@alex-mac, @jsakalys , thoughts on having the min.js/css files built into the bundles so we don't have to use cdns?

Reply:
great idea :) I'll look into it, still learning webpack myself